src/forest/method/h.ts

```diff
@@ -73,7 +88,6 @@ function createPropsOp<T, S>(
       group: leaf.root.leafOps[leaf.fullID].group,
     })
     leaf.root.leafOps[leaf.fullID].group.ops[opID] = op
-    const ctx = initCtx(value, leaf)
```

Comment:
Was this necessary for something specific? If you call initialization at the beginning, then [some data](https://github.com/effector/effector/pull/638/files#diff-df0554ad39e0813ff09cce9f39ab1f53bd84a9927f11a9309c96c6715941794fR90) may not make it [to processing](https://github.com/effector/effector/blob/0e20c015c0bf488affbaa5afcbebec86ed28e33a/src/forest/method/h.ts#L534)

Reply:
I had a exception "value used before initialization"
![telegram-cloud-photo-size-2-5325802100816460378-y](https://user-images.githubusercontent.com/5620073/161507115-18a0eeb2-b10c-4f4b-b639-a0b560ade19b.jpg)